suite-native/config/src/supportedNetworks.ts

```diff
@@ -22,20 +22,20 @@ export const mainnetsOrder: NetworkSymbol[] = [
 
 export const testnetsOrder: NetworkSymbol[] = ['test', 'regtest', 'tsep', 'tgor', 'tada', 'txrp'];
 
+export const supportedNetworkSymbols = Object.keys(networks) as NetworkSymbol[];
```

Comment:
Please use the same naming pattern: 
`deviceSupportedNetworkSymbols`
`accountImportSupportedNetworkSymbols`

so it's clear from the name what's difference between supported and importEnabled

Reply:
I used the name `supportedNetworkSymbols` these all the networks that we have. Device supported networks are filtered later in discoveryThunks and it is subset of these symbols and it is different for each device model.

Also the `importEnabledNetworkSymbols` use `supportedNetworkSymbols` in it's expression, so it would be strange if it would be filtering `deviceSupportedNetworkSymbols` as you suggest. I believe now it is clear that `importEnabledNetworkSymbols` are reflecting the import flow.